### DISCUSSION TITLE
[feature request] PDF cheatsheet of default keyboard shortcuts

### DISCUSSION THREAD

**Discussion by KrystalBiggs** [rxhanson/RectanglePro-Community#599] [General]
New Macpro m1 user here, coming from Windows:
I'd like to grab a 1 or 2-page cheatsheet PDF listing the default keyboard and action shortcuts
(keen to get a physical printout to help train myself)

**Comment by rxhanson** [comment]
Welcome to macOS! 

The default keyboard shortcuts are the same as for Rectangle, and you can grab the settings screenshot from https://rectangleapp.com. I think most people use the menu bar menu as a quick reference instead of a separate cheat sheet.

For the window throw, I think it's best to memorize the modifier keys (the default is control+command) and it should be fairly quick to acclimate to how it works.
